src/backend/toolmanager/utils.ts

```diff
@@ -0,0 +1,576 @@
+import * as axios from 'axios'
+import {
+  existsSync,
+  mkdirSync,
+  rmSync,
+  statSync,
+  unlinkSync
+} from 'graceful-fs'
+import { spawnSync, spawn } from 'child_process'
+import {
+  ProgressInfo,
+  State,
+  VersionInfo,
+  Type,
+  ToolVersionInfo,
+  Repositorys
+} from '../../common/types/toolmanager'
+import Store from 'electron-store'
+import {
+  logError,
+  logInfo,
+  LogPrefix,
+  logWarning
+} from '../../backend/logger/logger'
+import { getAvailableVersions, installVersion } from './toolmanager'
+import { heroicToolsPath } from '../../backend/constants'
+import { sendFrontendMessage } from '../main_window'
+
+const wineDownloaderInfoStore = new Store({
```

Comment:
we have a `src/frontend/helpers/electroStores.ts` file, we should keep them all there

Reply:
This is not the fronted, so should not work. We need something similiar in the backend. Maybe in another PR?

Reply:
yeah sure, not for this PR, I missed it was in the backend